README.md

```diff
@@ -1,62 +1,48 @@
-[![Conclave SDK](.github/assets/logo.png)][conclave_website]
+[![Conclave Core SDK](.github/assets/logo.png)][conclave_website]
 
-## Overview
-[Conclave SDK][docs] is an open source platform that makes working with SGX enclaves easy.
+The [Conclave Core SDK][docs] is an open source platform that lets you create SGX enclaves very easily. You can 
+write your enclave code in high-level languages such as Java, Kotlin and JavaScript.
 
-It is a toolkit for building **enclaves**, small pieces of software that are protected from attack 
-by the owner of the computer on which they run. 
-It is ideally suited to solving multi-party collaboration and privacy problems, 
+**Enclaves** are small pieces of software that are protected from attack by the owner of the computer on which they 
+run. They are ideally suited to solving multi-party collaboration and privacy problems, 
 but can also be used to secure your infrastructure against attack.
 
-Conclave SDK has been originally developed by [R3][r3_website], and it is has been made available 
-to the community, see the [license below](#license).
+The Conclave Core SDK is developed by [R3][r3_website], and has been made available to the [open source community](#license).
 
-### Documentation and tutorials
-Please refer to the [Conclave website for documentation and tutorials][docs].
-Sources of the documentation are under the directory [docs/](docs/docs).
+## How to use Conclave
+If you want to learn how to use Conclave and build enclaves, take a look at the
+[hello world example and tutorial][hello_world]. You can also refer to the [Conclave documentation][docs] and
+[API docs](https://docs.conclave.net/api/index.html).
 
-### Contributing
-We encourage you to contribute to Conclave. As a contributor please [follow these guidelines](#contributing.md).
+## Contributing
+We encourage you to contribute to Conclave. Have a read of our contribution [guidelines](CONTRIBUTING.md).
```

Comment:
Please rewrite 'Have a read of our contribution' as:

Please read our contribution...